src/components/DAO.tsx

```diff
@@ -23,15 +24,7 @@ class ArcDAO extends Component<Props, Entity, Data> {
         "Arc Config Missing: Please provide this field as a prop, or use the inference component."
       );
     }
-    return new Entity(address, arcConfig.connection);
-  }
-
-  protected async initialize(entity: Entity | undefined): Promise<void> {
```

Comment:
This has been removed since we are doing this fetchState on the `Component.tsx` - Thoughts?. @dOrgJelli

Reply:
Yep, makes sense!